src/zenml/post_execution/pipeline_run.py

```diff
@@ -156,6 +156,18 @@ def enable_cache(self) -> bool:
         enable_cache = self.pipeline_configuration["enable_cache"]
         return cast(bool, enable_cache)
 
+    @property
+    def enable_artifact_metadata(self) -> bool:
```

Comment:
a bit of a nitpick, but as a property I would expect it to not start with the verb, maybe `artifact_metadata_enabled` ?

Reply:
Somewhat reluctant to change this since the underlying attribute in the model is also called `enable_artifact_metadata`

Reply:
And we also have `enable_cache` as a property which would be the same issue I guess

Reply:
ok, ignored :D